src/Api/AdminConsole/Public/Models/Request/AssociationWithPermissionsRequestModel.cs

```diff
@@ -10,7 +10,8 @@ public CollectionAccessSelection ToCollectionAccessSelection()
         {
             Id = Id.Value,
             ReadOnly = ReadOnly.Value,
-            HidePasswords = HidePasswords.GetValueOrDefault()
+            HidePasswords = HidePasswords.GetValueOrDefault(),
+            Manage = Manage.GetValueOrDefault()
```

Comment:
I suggest that we only want to set this if `org.FlexibleCollections == true`. Otherwise I think it'll show up as a blank label in the web vault, because FC changes won't be enabled. And in any case, we don't want to let users start using FC features until their org has been migrated.

Reply:
After thinking about it, you should probably throw instead, just to provide unambiguous feedback to the user.